tests/test_schema.py

```diff
@@ -224,24 +224,18 @@ def test_semantic_mode_error(self):
     def test_semantic_error_policy_scope(self):
         data = {
             'policies': [
-                {'actions': [{'key': 'TagPolicyCompliance',
-                              'type': 'tag',
-                              'value': 'This resource should have tags following policy'}],
-                 'description': 'Identify resources which lack our accounting tags',
-                 'filters': [{'tag:Environment': 'absent'},
-                             {'tag:Service': 'absent'},
-                             {'or': [{'tag:Owner': 'absent'},
-                                     {'tag:ResponsibleParty': 'absent'},
-                                     {'tag:Contact': 'absent'},
-                                     {'tag:Creator': 'absent'}]}],
-                 'name': 'tagging-compliance-waf',
+                {'actions': [{'key': 'AES3000',
+                              'type': 'encryption',
+                              'value': 'This resource should have AES3000 encryption'}],
+                 'description': 'Identify resources which lack our outrageous cipher',
+                 'name': 'bogus-policy',
```

Comment:
This test seems to just be here to test that it errors, so I made it completely outrageous as I am sure that is what people used to think about being able to tag WAFs 😆